### DISCUSSION TITLE
Path to vendor when using separateVendor option

### DISCUSSION THREAD

**Discussion by svnldwg** [brefphp/bref#850] [Support] [answered]
Hi,

I am using the `separateVendor` option and had some trouble getting it to work. In my php file I had this statement:

`require_once __DIR__ . '/../vendor/autoload.php';`

which didn't work anymore when using the `separateVendor` option:

> PHP Fatal error:  require_once(): Failed opening required '/var/task/config/../vendor/autoload.php' ...

The way I got it to work was by changing the path to `/tmp/vendor/autoload.php`. Is that the recommended way of loading the autoloader when using the `separateVendor` option or am I missing something?

**Comment by mnapoli** [comment, marked as answer]
> Is that the recommended way of loading the autoloader when using the separateVendor option or am I missing something?

Yes it is, because the vendor zip is downloaded and unzipped in that directory. The reason for that is that only `/tmp` allows write-read, the rest of the filesystem is read-only.

That is probably something that could be improved in the documentation if it's not clear.

**Reply by svnldwg** [reply]
Thanks for the clarification, makes sense! I only found out about that path change by checking the code, which almost made me not use the feature. I will open a pull-request shortly to include that information in the documentation.
